_pages/speak.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Speak
+snake: rainbow
+permalink: /speak/
+
+---
+
+# Contents
+* [Call for Proposals](#cfp)
+* [Talk details](#talks)
+* [Mentors](#mentors)
+* [Helpful Tips](#helpful-tips)
+* [Financial support](#financial-support)
+
+## <a name="cfp"></a> Call for Proposals (CFP)
+
+We want you [to submit a talk proposal for PyCon AU](https://pretalx.com/pyconau2019/cfp)!
+
+We're happy to consider any talk which will be interesting and relevant to an audience of Python developers at any level of expertise or experience. 
+
+**The 2018 Call for Presentations (CFP) is open from Monday, April 3 [Anywhere On Earth (AoE)](https://en.wikipedia.org/wiki/Anywhere_on_Earth) to Monday, May 5 AoE.**
+
+
+## <a name="topics"></a> Talk details
+
+We're mostly looking for 30 minute talks on everything Python, open source, and tech-related. Want to share something cool? We want to hear from you! We welcome submissions from everybody, including those who've never spoken before or are relatively new to Python.
+
+PyCon AU attracks a diverse set of attendees so we're looking for a mix of talks which are beginner friendly as well as talks which provide real value to seasoned professionals. If you've got some prior speaking experience and you'd like to present on a more advanced or deeper technical topic, you might like to submit a [deep dive talk](/news/deep-dive-talks/). You get extra time and (as long as it's labelled clearly) you can assume the audience to have relevant prior knowledge and expertise.
+
+Talk topics for the main conference can be anything, but on the Friday of the conference we have four [specialist tracks](/attend/). For these tracks we are specifically looking for talks about Education, Django, Security and Privacy and Science and Data. Talks for all topics go through the same submission process but you can flag your talk for a specific track if you want.
```

Comment:
```suggestion
Talk topics for the main conference can be anything, but on the Friday of the conference we have four [specialist tracks](/attend/). For these tracks we are specifically looking for talks about Education, Django, Security and Privacy, and Science and Data. Talks for all topics go through the same submission process but you can flag your talk for a specific track if you want.
```

Oxford comma, since two of our Tracks have "and"s in them.